docs/notebooks/synthetic_data_uber_differential_privacy.ipynb

```diff
@@ -0,0 +1,1462 @@
+{
+  "nbformat": 4,
+  "nbformat_minor": 0,
+  "metadata": {
+    "accelerator": "GPU",
+    "colab": {
+      "name": "synthetic-data-uber-differential-privacy.ipynb",
+      "provenance": [],
+      "collapsed_sections": [],
+      "toc_visible": true
+    },
+    "kernelspec": {
+      "display_name": "Python 3",
+      "language": "python",
+      "name": "python3"
+    },
+    "language_info": {
+      "codemirror_mode": {
+        "name": "ipython",
+        "version": 3
+      },
+      "file_extension": ".py",
+      "mimetype": "text/x-python",
+      "name": "python",
+      "nbconvert_exporter": "python",
+      "pygments_lexer": "ipython3",
+      "version": "3.6.9"
+    }
+  },
+  "cells": [
+    {
+      "cell_type": "markdown",
+      "metadata": {
+        "id": "UTRxpSlaczHY"
+      },
+      "source": [
+        "# A differentially private, synthetic ride-share dataset\n",
+        "\n",
+        "This blueprint utilizes Gretel's premium SDKs to create a synthetic version of your own data. Our SDKs create automatic data validators to help ensure the data generated has the same semantics as the source data. Additionally, the SDKs do autmoatic header clustering to help maintain statistical relations between columns."
+      ]
+    },
+    {
+      "cell_type": "code",
+      "metadata": {
+        "id": "VEM6kjRsczHd"
+      },
+      "source": [
+        "%%capture\n",
+        "\n",
+        "!pip install -U gretel-client gretel-synthetics pandas"
+      ],
+      "execution_count": 1,
+      "outputs": []
+    },
+    {
+      "cell_type": "code",
+      "metadata": {
+        "id": "ZQ-TmAdwczHd",
+        "colab": {
+          "base_uri": "https://localhost:8080/"
+        },
+        "outputId": "45d3bb69-d336-4e63-9662-2a0601a54c02"
+      },
+      "source": [
+        "# Load your Gretel API key. You can acquire this from the Gretel Console \n",
+        "# @ https://console.gretel.cloud\n",
+        "\n",
+        "from getpass import getpass\n",
+        "import pandas as pd\n",
+        "from gretel_client import configure_session, ClientConfig\n",
```

Comment:
You can replace the auth step with the latest call, which automatically wraps getpass and caches credentials.

```
from gretel_client import configure_session

configure_session(api_key="prompt", validate=True, cache=True)
```